src/parse.py

```diff
@@ -30,6 +31,14 @@ def open_json_file(filename):
         raise ValueError('{} is not valid JSON.'.format(filename))
 
 
+def validate_object_keys(keys: Union[Tuple[str], List[str]], test_json):
+    for key in keys:
+        if key in test_json.keys():
+            continue
+        raise Exception('Key is missing')
+    return True
```

Comment:
what will  be returned if empty array is passed as a `keys` param?

Reply:
validate_object_keys([],'{q=1}') it's not valid Json format, '{q=1}' this is String

Reply:
need to add a check for an empty key list?

Reply:
test_validate_object_keys_raise_exception_with_wrong_key is OK yet